predict.py

```diff
@@ -48,14 +33,14 @@ def load_model(training_dir, device):
     return model.eval(), num_classes
 
 
-def get_dataset_properties(dataset_ids):
+def get_dataset_labels(dataset_ids):
```

Comment:
No need to use properties since the current constraint is that we can use only 1 dataset and only 1 category.